docs/modeling/blackbody_deprecated.rst

```diff
@@ -0,0 +1,124 @@
+
+BlackBody1D class
+-----------------
+
+The equivalency between the
+:class:`~astropy.modeling.physical_models.BlackBody` and
+:class:`~astropy.modeling.blackbody.BlackBody1D`
+is shown in the plot below.
+
+.. plot::
+    :include-source:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astropy.modeling.blackbody import BlackBody1D
+    from astropy.modeling.models import BlackBody
+    import astropy.units as u
+
+    wavelengths = np.logspace(np.log10(1000), np.log10(3e4), num=1000) * u.AA
+
+    # blackbody parameters
+    temperature = 10000 * u.K
+
+    # BlackBody1D is normalized by default to have a bolometric flux of 1
+    bb1d = BlackBody1D(temperature)
+
+    # default results from BlackBody same units as BlackBody1D
+    bb = BlackBody(temperature)
+
+    # plot the results from both methods
+    fig, ax = plt.subplots(ncols=1, figsize=(4.0, 4.0))
+
+    ax.plot(wavelengths, bb1d(wavelengths), label="BlackBody1D", linewidth=6, alpha=0.5)
+
+    # divide by bolometric flux to get equivalent of BlackBody1D
+    ax.plot(wavelengths, bb(wavelengths)/bb.bolometric_flux, label="BlackBody")
+
+
+    ax.set_xlabel(r"$\lambda$ [{}]".format(wavelengths.unit))
+    ax.set_ylabel(r"$Flux$ [{}]".format(bb(wavelengths).unit))
+    ax.set_xscale("log")
+    ax.set_yscale("log")
+    ax.legend(loc="best")
+
+    plt.tight_layout()
+    plt.show()
+
+blackbody functions (deprecated)
+--------------------------------
+
+Prior to astropy v4.0, two simple functions were provided to compute
+a blackbody in frequency and wavelength based units -
+:func:`~astropy.modeling.blackbody.blackbody_nu`
+and :func:`~astropy.modeling.blackbody.blackbody_lambda`.
+These functions are deprecated as they same
+results can be obtained using the
+:class:`~astropy.modeling.physical_models.BlackBody` model.
+
+Equivalent results between
+and :class:`~astropy.modeling.physical_models.BlackBody` and the
+func:`~astropy.modeling.blackbody.blackbody_nu` and
+func:`~astropy.modeling.blackbody.blackbody_lambda` is shown in the plot below.
+
+.. plot::
+    :include-source:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astropy.modeling.blackbody import blackbody_lambda, blackbody_nu
+    from astropy.modeling.models import BlackBody
+    import astropy.units as u
+
+    wavelengths = np.logspace(np.log10(1000), np.log10(3e4), num=1000) * u.AA
+
+    # blackbody parameters
+    temperature = 10000 * u.K
+
+    # provides the results in ergs/(cm^2 Hz s sr)
+    spectrum_nu = blackbody_nu(wavelengths, temperature)
+
+    # provides the result in ergs/(cm^2 A s sr)
+    spectrum_lam = blackbody_lambda(wavelengths, temperature)
+
+    # default results from BlackBody same units as blackbody_nu
+    bb_nu = BlackBody(temperature)
+
+    # use scale to change results units for Blackbody to match blackbody_lambda
+    bb_lam = BlackBody(temperature, scale=1.0 * u.erg / (u.cm ** 2 * u.AA * u.s * u.sr))
```

Comment:
Interesting... I wouldn't have thought of using `scale` in that way. I would have done something more involved, like:
```python
 bb_nu(wavelengths).to(u.erg / (u.cm ** 2 * u.AA * u.s * u.sr), u.spectral_density(wavelengths))
```
So, thanks for including this example!

Reply:
Using scale this way make it "easier" for the user *and* (after my latest commit) also means that the scale units can be used to set the output units even during fitting.  This can't be done using the return_units functionality of modeling as spectral_density equivalencies is needed and this isn't support by return_units at this point.

Reply:
What you would have done works as well of course.